core/google/cloud/_helpers.py

```diff
@@ -22,27 +22,24 @@
 
 import calendar
 import datetime
-import json
 import os
 import re
-import socket
 from threading import local as Local
 
+import google.auth
 from google.protobuf import timestamp_pb2
-try:
-    from google.appengine.api import app_identity
-except ImportError:
-    app_identity = None
-try:
+import google_auth_httplib2
+
+try:  # pragma: NO COVER
     import grpc
-except ImportError:  # pragma: NO COVER
+    from google.auth.transport.grpc import AuthMetadataPlugin
```

Comment:
So now that I understand

1. Why not have the test import `grpc`?
1. `from google.auth.transport.grpc import AuthMetadataPlugin` is the only line that doesn't get covered (the first failed import does and then the `except` block all gets covered)